### PR TITLE
Profile image user

### PR DESCRIPTION
**Pull request by nathaliedpettersson**
These are all my changes since yesterday regarding creating user. 

Fixed img-problem so we now send the string of the img-source to database. 

Also cleaned up in the code and fixed errors in our files

### CODE REVIEW

File: src/modules/firebaseApp.ts [hunk 1]
```diff
@@ -29,14 +29,14 @@ const dbFoodForum = ref(db, '/SNLApp/Forum/food/');
 const dbWineForum = ref(db, '/SNLApp/Forum/wine/');
 
 
-export function getBeerInDb(callback: (topic:Topic) => void) {
+export function getBeerInDb(callback: (topic: Topic) => void) {
```
Comment (Takikato09) [inline src/modules/firebaseApp.ts:32]: was there an error with the code because of the space?

**Review by Takikato09 — COMMENTED** [review]
Good job Natta. Just one thing: Does the space in topic:Topic made some errors in the code?

**Review by SaraM83 — APPROVED** [review]
Great work Natta, can we add pics to the existing users by hand so I can test displaying it on the profile page?

File: src/modules/firebaseApp.ts (hunk 1 above)
Comment (nathaliedpettersson) [reply src/modules/firebaseApp.ts:32]: Hmm, I haven't changed that? Haha, that's weird that it says I did something on that line. 
Maybe I accidentally had the mouse there at one point and did something?! Anyway, it wasn't intentional so I don't know if it made a difference or not :angel:

Comment (Takikato09) [reply src/modules/firebaseApp.ts:32]: haha alright! was just wondering, maybe VS code automatically does it :) anyways, really good job!!

**Pull request merged**